Resources/Prototypes/Entities/Mobs/NPCs/lavaland.yml

```
@@ -1,4 +1,116 @@
 ﻿- type: entity
+  name: magma hand
+  id: MobMagmaHand
+  parent: [ LavalandMobBase, MobCombat ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)